### DISCUSSION TITLE
Custom stuff like grid?

### DISCUSSION THREAD

**Discussion by jenstornell** [tailwindlabs/tailwindcss#1616] [General]
I'm new to Tailwind. In fact, I've seen like 10 videos about it but not yet done anything with it. Before I make my mega project I want to know that I don't need to start all over again without Tailwind later.

## Grid

Now I talk about css `display:grid;`. What I can see it's not supported by Tailwind, at least not any utility classes for it.

I work a lot with grids as it's very powerful. Switching to tailwind it looks like I will use Flexbox more?

I've seen custom css can be added to the css file. Is that what we do when we need to use grid?

**Comment by Apostolique** [comment]
You can see all the display utilities here: https://tailwindcss.com/docs/display/

`grid` is what you want.

**Comment by adamwathan** [comment]
We definitely support grid, look for the "Grid" section in the documentation sidebar:

https://tailwindcss.com/docs/grid-template-columns
